README.md

```diff
@@ -53,6 +53,10 @@ pip install -U pip
 # Clone and Install the QEfficient Repo.
 pip install git+https://github.com/quic/efficient-transformers
 
+# Or build wheel package using the below command.
+python -m build --wheel --outdir dist
+pip install dist/QEfficient-0.0.1.dev0-py3-none-any.whl
```

Comment:
We can specify the installation of `build` and `wheel` packages here.

Reply:
installation updated here.